cmd/influxd/server_integration_test.go

```diff
@@ -1905,6 +1905,94 @@ func Test_ServerOpenTSDBIntegration_BadData(t *testing.T) {
 	}
 }
 
+func Test_ServerOpenTSDBIntegrationHTTPSingle(t *testing.T) {
```

Comment:
There is a lot of boilerplate here. Do you think you could combine the tests? If you use a different series for each test, you could craft queries that wouldn't overlap.